### DISCUSSION TITLE
[Question]: camera.cameraname_object is gone after update to 0.13

### DISCUSSION THREAD

**Discussion by QBANIN** [blakeblackshear/frigate#9902] [Ask A Question]
### What is your question:

Hi!

After update to 0.13 previously existing entity camera.cameraname_object  in Home Assistant (i.e. camera.front_person) are gone. Is there any way to bring it back? Is that a matter of Frigate itself or HA integration?

**Comment by NickM-27** [comment]
This is in the release notes, it has moved to image.cameraname_object
